numba/tests/test_pythonapi.py

```diff
@@ -78,5 +79,45 @@ def test_PyBytes_AsStringAndSize(self):
         self.assertEqual(obj, b'hello\x00world')
 
 
+class PythonAPIEmptyArgs(unittest.TestCase):
+    def test_empty_args(self):
+        def callme(**kwargs):
+            print("callme", kwargs)
+
+        @intrinsic
+        def py_call(tyctx):
+            def codegen(context, builder, sig, args):
+                pyapi = context.get_python_api(builder)
+                gil = pyapi.gil_ensure()
+
+                num = pyapi.long_from_long(
```

Comment:
I'm guessing a bit, but maybe this should have been

```suggestion
                num = pyapi.long_from_longlong(
```

because I think `long` might be an `i32` on Windows, causing the current test failure.

Reply:
ahh yea, right, I did fix it before but here I forgot